cookbook/direnv.md

```diff
@@ -22,6 +22,9 @@ $env.config = {
       }
 
       direnv export json | from json | default {} | load-env
+      if 'ENV_CONVERSIONS' in ($env | columns) and 'PATH' in ($env.ENV_CONVERSIONS | columns) {
```

Comment:
I don't think you need the `| columns` parts
```suggestion
      if 'ENV_CONVERSIONS' in $env and 'PATH' in $env.ENV_CONVERSIONS {
```

Reply:
Nice!